src/components/TodosFilter.tsx

```diff
@@ -0,0 +1,49 @@
+import { useContext } from 'react';
+import { Status } from '../types/todo';
+import { TodosContext } from './TodosContext';
+
+type Props = {};
+
+export const TodosFilter: React.FC<Props> = () => {
+  const {
+    selectedStatus, setSelectedStatus,
+  } = useContext(TodosContext);
+
+  const handleStatusClick = (status: Status) => {
+    setSelectedStatus(status);
+  };
+
+  return (
+    <ul className="filters" data-cy="todosFilter">
+      <li>
+        <a
+          href="#/"
+          className={`${selectedStatus === Status.all ? 'selected' : ''}`}
+          onClick={() => handleStatusClick(Status.all)}
+        >
+          All
+        </a>
+      </li>
+
+      <li>
+        <a
+          href="#/active"
+          className={`${selectedStatus === Status.active ? 'selected' : ''}`}
```

Comment:
```suggestion
          className={selectedStatus === Status.active ? 'selected' : ''}
```
you can simplify it or use the classnames package